### PR TITLE
fix: django shortcuts styling

### PR DESCRIPTION
**Pull request by fsbraun**
This PR reintroduces the styling for django-shortcuts which were deleted with their icons. The icons are not reintroduced and django-shortcuts will use its own icons also with djangocms-admin-style.

Related resources
* #480

### CODE REVIEW

**Comment by DmytroLitvinov** [comment]
Waiting for new release :)

File: README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
-======================
 django CMS Admin Style
 ======================
 
 |pypi| |build| |coverage|
```
Comment (marksweb) [inline README.md:4]: The buttons break with the file format change.

**Review by marksweb — CHANGES_REQUESTED** [review]
`isort` isn't happy about something here, but I'm not entirely sure what.

File: README.md (hunk 1 above)
Comment (fsbraun) [reply README.md]: Yeah. I hoped I could just copy and past the tables but now I converted them to rst...

**Comment by fsbraun** [comment]
@marksweb There was a superfluous empty line in `setup.py`. Removed.

**Comment by marksweb** [comment]
@fsbraun that explains how I looked over it 😂

**Review by marksweb — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -4,7 +4,8 @@ django CMS Admin Style
 
 |pypi| |build| |coverage|
 
-Adds pretty CSS styles for the django CMS admin interface.
+Adds pretty CSS styles for the django CMS admin interface. Supports optional`
+django-shortcuts package.
```
Comment (marksweb) [inline README.rst:8]:
```suggestion
``django-shortcuts`` package.
```

File: README.rst [hunk 3]
```diff
@@ -4,7 +4,8 @@ django CMS Admin Style
 
 |pypi| |build| |coverage|
 
-Adds pretty CSS styles for the django CMS admin interface.
+Adds pretty CSS styles for the django CMS admin interface. Supports optional`
```
Comment (marksweb) [inline README.rst:7]:
```suggestion
Adds pretty CSS styles for the django CMS admin interface. Supports optional
```

**Review by marksweb — CHANGES_REQUESTED** [review]
Just spotted syntax issues in the readme

**Comment by fsbraun** [comment]
@DmytroLitvinov Please check pypi.

**Comment by DmytroLitvinov** [comment]
Thank you, guys 🙌

**Pull request merged**